articles/concepts/Glossary.md

```diff
@@ -12,31 +12,33 @@ uid: microsoft.quantum.glossary
-|Tuple|Comma separated types grouped together via parenthesis. |
-|User-defined type|Collection of built-in or previously defined types that may be referred to as a single unit.|
-
+|<a id="adjoint"></a>Adjoint|The complex conjugate transpose of an operation. For operations that implement a unitary operator, the adjoint is the inverse of the operation. Adjoints are indicated by dagger symbol, for example the adjoint of vector $v$ is $v^\dagger$. For more information, see [Adjoint](xref:microsoft.quantum.language.file-structure#adjoint).|
+|<a id="bell-state"></a>Bell state|A maximally entangled quantum state of two qubits. Also known as an [EPR pair](xref:microsoft.quantum.glossary#epr)|
+|<a id="callable"></a>Callable|In the Q# language, operations and functions are collectively known as *callables*. For more information, see [Operation and function types](xref:microsoft.quantum.language.type-model#operation-and-function-types).|
+|<a id="clifford"></a>Clifford group|The set of operations that occupy the octants of the Bloch sphere and effect permutations of the Pauli operators. These include: `X`, `Y`, `Z`, `H` and `S`|
+|<a id="controlled"></a>Controlled|A quantum operation that takes one or more qubits as enablers for the target operation. For more information, see [Controlled](xref:microsoft.quantum.language.type-model#controlled).|
+|<a id="dirac"></a>Dirac Notation|A shorthand representation of quantum states, for example $$\frac{1}{\sqrt{2}}\begin{bmatrix} 1 \\\\  1 \end{bmatrix}=H\ket{0} = \ket{+}$$. For more information, see [Dirac Notation](xref:microsoft.quantum.concepts.dirac).|
+|<a id="eigenvalue"></a>Eigenvalue| The factor by which the magnitude of an [eigenvector](xref:microsoft.quantum.glossary#eigenvector) is changed by a given transformation. For more information, see [Advanced matrix concepts](xref:microsoft.quantum.concepts.matrix-advanced).|
+|<a id="eigenvector"></a>Eigenvector|A vector whose direction is unchanged by a given transformation and whose magnitude is changed by a factor corresponding to that vector's [eigenvalue](xref:microsoft.quantum.glossary#eigenvalue). For more information, see [Advanced matrix concepts](xref:microsoft.quantum.concepts.matrix-advanced).|
+|<a id="epr"></a>EPR pair|A maximally entangled quantum state of two qubits. Also known as a [Bell state](xref:microsoft.quantum.glossary#bell-state)|
+|<a id="evolution"></a>Evolution|How a quantum state changes over time. For more information, see [Matrix exponentials](xref:microsoft.quantum.concepts.matrix-advanced#matrix-exponentials).|
+|<a id="function"></a>Function|A purely classical (non-quantum) routine in the Q# language. For more information, see [Operation and function types](xref:microsoft.quantum.language.type-model#operation-and-function-types).|
```

Comment:
Notably, not all purely classical routines are functions! Random number sampling, machine environment queries (e.g.: those in Microsoft.Quantum.Environment) and other nondeterminstic classical logic is also represented by operations.